### ISSUE TITLE
[Chore] Release on npm before github ?

### ISSUE THREAD

**Issue by noraj** [FortAwesome/Font-Awesome#16354]
How comes that there is a 5.13.0 release on npmjs before the release it's on github ?

![image](https://user-images.githubusercontent.com/16578570/77352827-1d972280-6d40-11ea-8d15-ade77fc34494.png)

![image](https://user-images.githubusercontent.com/16578570/77352843-2687f400-6d40-11ea-8412-e907e2e82e3e.png)

![image](https://user-images.githubusercontent.com/16578570/77352851-2be53e80-6d40-11ea-9e4f-53633113ad30.png)

So renovate is proposing me an update but no changelog is available yet because it's a ghost release.

![image](https://user-images.githubusercontent.com/16578570/77352884-3dc6e180-6d40-11ea-9d7d-cdea8906ccfe.png)

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

> How comes that there is a 5.13.0 release on npmjs before the release it's on github ?

This is not the source repository of the release, let's ask @robmadole if it is possible to perform push on the free repository here before releasing a new version in the future

**Comment by robmadole** [comment]
The release takes a while and I have lots of steps. I try and pick the most efficient way based on all the moving parts. Right now releasing the the npm packages actually frees up other updates that need to be made.

**Comment by noraj** [comment]
Ok, I guess there is a complex workflow involved.

**Comment by robmadole** [comment]
@noraj it is. It takes most of the day to release a version of Font Awesome. Is this dependency order causing you a headache? I'd like to understand what happened a little better.

**Comment by tagliala** [comment]
>  I'd like to understand what happened a little better.

I may think that some automation tools like [Dependabot](https://github.com/dependabot), after detecting a new version, try to get the changelog  in the repository and did not find them

It didn't happen to me, but take a look at the first comment in this PR:  diowa/ruby2-rails6-bootstrap-heroku#57

**Comment by robmadole** [comment]
Ok. I think I can switch the order that these things happen in without much trouble. Thanks @tagliala

**Comment by noraj** [comment]
Yes, exactly RenovateBot notify me about a new update but no changelog is available so when I receive my notification I can't know if it is a good option to update or not.
